docs/sql/statements/alter-table.rst

```diff
@@ -188,6 +188,12 @@ Can be used to drop a column from a table.
 
 It's possible to drop multiple columns at once.
 
+.. NOTE::
+
+    All constraints on the dropped column, including constraints on sub-columns
+    of the dropped column (if its type is :ref:`type-object`), are dropped as
+    well.
```

Comment:
I'm not completely sure if this is needed as for me it seems logically that the constraints on the column are dropped together with the column. Leaving the constraints without the column wouldn't make any sense or?

Reply:
yep, it's fine, I can remove, thx!